app/javascript/components/util/InfoTooltip.js

```diff
@@ -141,7 +141,8 @@ const TOOLTIP_TEXT = {
   blockedSMS: (
     <div>
       The owner of this phone number has texted &quot;STOP&quot; in response to a Sara Alert text message. This means that this phone number cannot receive text
-      messages from Sara Alert and should not be assigned SMS Preferred Reporting Methods unless the user replies &quot;START&quot; to a Sara Alert message.
+      messages from Sara Alert and should not be assigned SMS Preferred Reporting Methods unless the monitoree replies &quot;START&quot; to a Sara Alert
```

Comment:
maybe we can wrap the entire string like:
```
{'The owner of this phone number has texted "STOP" in response to a Sara Alert text message. This means that this phone number cannot receive text messages from Sara Alert and should not be assigned SMS Preferred Reporting Methods unless the monitoree replies &quot;START&quot; to a Sara Alert'}
```
to avoid having to use `&quot;` making it easier to read?

Reply:
I'm not necessarily against that, but there are a lot of other places in this file where we use `&quot;` and `&apos;`. Think I should change them all? Or just this one? Or just leave it?

Reply:
i think i'll leave these as is. Unless @mayerm94 or @sgober has strong opinions.

Reply:
I'd leave it as-is, using the quotes makes ESLint fail.